### PR TITLE
Add page about WASM summit.

### PR DESCRIPTION
**Pull request by ntoll**
Hi folks :wave: 

I've added the blurb for the WASM summit via the page created in this PR.

~~**TODO**: Someone (who?) will need to add this page to the menu system so it's easy to find from the top level menu.~~

Happy to change anything or adjust according to feedback. It's basically a cut and paste of last year's page. :wink:

### CODE REVIEW

**Comment by hypha** [comment]
@ntoll you can change this file to add it on the menu thing: src/data/links.json

**Comment by ntoll** [comment]
@hypha coolio... I didn't know where/how to do this. Will push an update in a moment.

**Comment by ntoll** [comment]
@hypha OK... so I can't login to Vercel to see the site preview. Can you add me via my GitHub account please..?

**Comment by hypha** [comment]
> @hypha OK... so I can't login to Vercel to see the site preview. Can you add me via my GitHub account please..?

@ntoll might be better to ping a boffin on discord to give you permission! <3

File: src/content/pages/wasm.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: WebAssembly Summit
+subtitle: A summit to bring together maintainers and users of the Python with WebAssembly, to discuss the state of this ecosystem, existing challenges and ongoing work.
```
Comment (cybit) [inline src/content/pages/wasm.mdx:3]:
```suggestion
subtitle: A summit to bring together maintainers and users of Python with WebAssembly, to discuss the state of this ecosystem, existing challenges, and ongoing work.
```
I expect "the Python with WebAssembly" not to be a specific phrase.

File: src/content/pages/wasm.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+# WebAssembly Summit
+
+* **When**: Tuesday, July 9th.
+* **Where**: Prague Congress Centre (PCC), Room TBC
```
Comment (cybit) [inline src/content/pages/wasm.mdx:9]:
```suggestion
* **Where**: Prague Congress Centre (PCC), Room Club D
```
I'll just have the audacity to lock it in.

File: src/content/pages/wasm.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
```
Comment (cybit) [inline src/content/pages/wasm.mdx:10]:
```suggestion
* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
```
I don't think Conference is allowed as this is Wednesday to Friday only. @thepetk , @artcz could we please get an official answer?

File: src/content/pages/wasm.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+This summit aims to bring together maintainers and users of the Python with WebAssembly, to discuss the state of this ecosystem, existing challenges and ongoing work.
```
Comment (cybit) [inline src/content/pages/wasm.mdx:12]:
```suggestion
This summit aims to bring together maintainers and users of Python with WebAssembly, to discuss the state of this ecosystem, existing challenges, and ongoing work.
```
See above.

**Review by Vaibhavs10 — COMMENTED** [review]
Thanks @ntoll 🤗 - just came here to say that I tested the form and it works well (please ignore the first entry in it haha)

**Comment by ntoll** [comment]
@Vaibhavs10 too late! You've been booked for the opening WASM keynote. :stuck_out_tongue:

File: src/content/pages/wasm.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Read about last year's summit [here](https://github.com/psf/webassembly/blob/main/meetings/2023/webassembly-summit-europython.md).
+
+
+## Agenda
+
+* 9:00: Meet and greet (many of us won’t know each other) + unconference-y post-it based organisation.
+* 9:30: Presentations (30mins each)
+* 11:00: Coffee
+* 11:20: Presentations
```
Comment (cybit) [inline src/content/pages/wasm.mdx:22]:
```suggestion
* 11:15: Presentations
```
Feel free to keep your own schedule. The official coffee break is anounced to be from 11:00 to 11:15 .

File: src/content/pages/wasm.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+* 13:00: Lunch at the PCC (included)
```
Comment (cybit) [inline src/content/pages/wasm.mdx:23]: Current schedule has lunch from 12:45 to 13:45 .

File: src/content/pages/wasm.mdx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* 14:00: Unconference-y activities (discussions, hacks, ad hoc tutorials etc…)
+* 16:00: Round-up / plenary session for feedback and organising next steps.
+* 18:30-ish: Ad hoc dinner plans.
+
+## Registration
+
+You need to have a valid EuroPython in-person ticket ([Conference, Tutorial or Combined](https://ep2024.europython.eu/tickets#ticket-types)) to participate. Purchase a EuroPython ticket [here](https://tickets.europython.eu/), if you haven't already.
```
Comment (cybit) [inline src/content/pages/wasm.mdx:30]:
```suggestion
You need to have a valid EuroPython in-person ticket ([Tutorial or Combined](https://ep2024.europython.eu/tickets#ticket-types)) to participate. Purchase a EuroPython ticket [here](https://tickets.europython.eu/), if you haven't already.
```

See above.

File: src/content/pages/wasm.mdx (hunk 3 above)
Comment (ntoll) [reply src/content/pages/wasm.mdx:10]: Well, I just copied the advice I was given for last year. Remember, this is a technical summit **for the WASM community**, not a tutorial. My vote is to cast the net as wide as possible.

File: src/content/pages/wasm.mdx (hunk 5 above)
Comment (ntoll) [reply src/content/pages/wasm.mdx:22]: I want to ensure we're in sync. :+1:

File: src/content/pages/wasm.mdx (hunk 7 above)
Comment (ntoll) [reply src/content/pages/wasm.mdx:30]: See comment above about casting a wide net.

**Comment by ntoll** [comment]
@cybit thank you **so much** for your review. Really appreciate it. I've merged in most of the changes you suggest (even though you use an Oxford comma :stuck_out_tongue: ). I believe as wider pool of attendees is best so would prefer to allow folks with just a conference ticket into the community organised WASM summit.

File: src/content/pages/wasm.mdx (hunk 3 above)
Comment (cybit) [reply src/content/pages/wasm.mdx:10]: Any ticket is fine. Has been confirmed.

**Pull request merged**